### PR TITLE
Remove typo second domains directory in default domainHome

### PR DESCRIPTION
**Pull request by rjeberhard**

### CODE REVIEW

**Review by ddsharpe — APPROVED** [review]

File: documentation/domains/Domain.json [hunk 1]
```diff
@@ -374,7 +374,7 @@
           "$ref": "#/definitions/ServerService"
         },
         "domainHome": {
-          "description": "The directory containing the WebLogic domain configuration inside the container. Defaults to /shared/domains/domains/\u003cdomainUID\u003e if `domainHomeSourceType` is PersistentVolume. Defaults to /u01/oracle/user_projects/domains/ if `domainHomeSourceType` is Image. Defaults to /u01/domains/\u003cdomainUID\u003e if `domainHomeSourceType` is FromModel.",
+          "description": "The directory containing the WebLogic domain configuration inside the container. Defaults to /shared/domains/\u003cdomainUID\u003e if `domainHomeSourceType` is PersistentVolume. Defaults to /u01/oracle/user_projects/domains/ if `domainHomeSourceType` is Image. Defaults to /u01/domains/\u003cdomainUID\u003e if `domainHomeSourceType` is FromModel.",
```
Comment (robertpatrick) [inline documentation/domains/Domain.json:377]: The sentence "Defaults to /u01/oracle/user_projects/domains/ if `domainHomeSourceType` is Image" needs to be modified to append the domainUID directory name.

File: documentation/domains/Domain.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ The specification of the operation of the WebLogic domain. Required.
...
 | `configuration` | [Configuration](#configuration) | Models and overrides affecting the WebLogic domain configuration. |
 | `dataHome` | string | An optional directory in a server's container for data storage of default and custom file stores. If `dataHome` is not specified or its value is either not set or empty, then the data storage directories are determined from the WebLogic domain configuration. |
-| `domainHome` | string | The directory containing the WebLogic domain configuration inside the container. Defaults to /shared/domains/domains/<domainUID> if `domainHomeSourceType` is PersistentVolume. Defaults to /u01/oracle/user_projects/domains/ if `domainHomeSourceType` is Image. Defaults to /u01/domains/<domainUID> if `domainHomeSourceType` is FromModel. |
+| `domainHome` | string | The directory containing the WebLogic domain configuration inside the container. Defaults to /shared/domains/<domainUID> if `domainHomeSourceType` is PersistentVolume. Defaults to /u01/oracle/user_projects/domains/ if `domainHomeSourceType` is Image. Defaults to /u01/domains/<domainUID> if `domainHomeSourceType` is FromModel. |
```
Comment (robertpatrick) [inline documentation/domains/Domain.md:25]: Same comment as before regarding the default directory name for Image.

File: documentation/domains/index.html [hunk 3]
```diff
@@ -1295,7 +1295,7 @@
           "$ref": "#/definitions/ServerService"
         },
         "domainHome": {
-          "description": "The directory containing the WebLogic domain configuration inside the container. Defaults to /shared/domains/domains/\u003cdomainUID\u003e if `domainHomeSourceType` is PersistentVolume. Defaults to /u01/oracle/user_projects/domains/ if `domainHomeSourceType` is Image. Defaults to /u01/domains/\u003cdomainUID\u003e if `domainHomeSourceType` is FromModel.",
+          "description": "The directory containing the WebLogic domain configuration inside the container. Defaults to /shared/domains/\u003cdomainUID\u003e if `domainHomeSourceType` is PersistentVolume. Defaults to /u01/oracle/user_projects/domains/ if `domainHomeSourceType` is Image. Defaults to /u01/domains/\u003cdomainUID\u003e if `domainHomeSourceType` is FromModel.",
```
Comment (robertpatrick) [inline documentation/domains/index.html:1298]: Same comment as above.

File: kubernetes/crd/domain-crd.yaml [hunk 4]
```diff
@@ -283,8 +283,8 @@ spec:
                 type: object
               domainHome:
                 description: The directory containing the WebLogic domain configuration
-                  inside the container. Defaults to /shared/domains/domains/<domainUID>
-                  if `domainHomeSourceType` is PersistentVolume. Defaults to /u01/oracle/user_projects/domains/
+                  inside the container. Defaults to /shared/domains/<domainUID> if
+                  `domainHomeSourceType` is PersistentVolume. Defaults to /u01/oracle/user_projects/domains/
```
Comment (robertpatrick) [inline kubernetes/crd/domain-crd.yaml:287]: same comment about the Image default location.

File: operator/src/main/java/oracle/kubernetes/weblogic/domain/model/DomainSpec.java [hunk 5]
```diff
@@ -61,7 +61,7 @@
    */
   @Description(
       "The directory containing the WebLogic domain configuration inside the container."
-          + " Defaults to /shared/domains/domains/<domainUID> if `domainHomeSourceType` is PersistentVolume."
+          + " Defaults to /shared/domains/<domainUID> if `domainHomeSourceType` is PersistentVolume."
           + " Defaults to /u01/oracle/user_projects/domains/ if `domainHomeSourceType` is Image."
```
Comment (robertpatrick) [inline operator/src/main/java/oracle/kubernetes/weblogic/domain/model/DomainSpec.java:65]: Same comment.

**Review by robertpatrick — CHANGES_REQUESTED** [review]

File: documentation/domains/Domain.json (hunk 1 above)
Comment (rjeberhard) [reply documentation/domains/Domain.json:377]: What's listed is the correct value when the domainHomsSourceType is image. The reason is that we have customers who want to use the same image for different domains, which would have different domainUIDs. Therefore, we accept the domainHome in this case either being at or just above the actual domainHome value.

Comment (rjeberhard) [reply documentation/domains/Domain.json]: I've updated the text. Let me know what you think.

File: operator/src/main/java/oracle/kubernetes/weblogic/domain/model/DomainSpec.java (hunk 5 above)
Comment (rjeberhard) [reply operator/src/main/java/oracle/kubernetes/weblogic/domain/model/DomainSpec.java:68]: This one is the source of truth and the others are generated as part of the build.

**Review by robertpatrick — APPROVED** [review]

**Comment by robertpatrick** [comment]
Working on getting these content changes into a current PR.  Will close this one once the new PR is ready.

**Pull request closed without merging**